scripts/validations.py

```diff
@@ -0,0 +1,125 @@
+import itertools
+import json
+import sys
+
+ACTION_CONTEXT = 'context'
+ACTION_COUNT = 'count'
+ACTIONS = (ACTION_CONTEXT, ACTION_COUNT)
+
+UNLISTED_ADDONS = set()
+
+
+def parse_validations(results):
+    return (json.loads(result) for result in results)
+
+
+def unlisted_validations(results, unlisted_addons=None):
+    if unlisted_addons is None:
+        unlisted_addons = UNLISTED_ADDONS
+    return (result
+            for result in results
+            if (not result['metadata'].get('listed', True)
+                or result['metadata'].get('id') in unlisted_addons))
```

Comment:
I can explain at least some of the results.

* Some add-ons have changed from listed to unlisted, and most developers seem to upload their initial submission the first time *before* clicking the unlisted checkbox, which means they should have both unlisted and listed validation results.
* Some results have probably been submitted to the unlisted validator, which developers need to choose the validation type for each time.
* 100% of validations with listed as `True` or `None` must have a matching unlisted add-on, because you're filtering through `unlisted_validations` before counting, so they'd be filtered out otherwise.